DiaHelperApplication/DiaHelperApplication/CustomCells/HomeVcMealTableViewCell.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  HomeVcMealTableViewCell.swift
+//  DiaHelperApplication
+//
+//  Created by Petya Damyanova on 11.12.23.
+//
+
+import Foundation
+
+import UIKit
+
+class HomeVcMealTableViewCell: UITableViewCell {
+    let timestampLabel = UILabel()
+    let bloodSugarLabel = UILabel()
+    let insulinDoseLabel = UILabel()
+    let carbsIntakeLabel = UILabel()
+    let foodTypeLabel = UILabel()
+    
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        addConstraints()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    private func addConstraints() {
```

Comment:
This layout does not scale with display sizes. StackView can be setup to have children with the same or proportional sizes.